src/assets/scss/_tabs.scss

```diff
@@ -298,181 +337,161 @@
 }
 
 .nav-tabs-fillup {
-  -webkit-backface-visibility: hidden;
-  -moz-backface-visibility: hidden;
   backface-visibility: hidden;
-  -webkit-perspective: 1000;
-  -moz-perspective: 1000;
   perspective: 1000;
+
   > li {
     overflow: hidden;
+
     > a {
-      -webkit-backface-visibility: hidden;
-      -moz-backface-visibility: hidden;
       backface-visibility: hidden;
-      -webkit-transition: color 0.3s ease 0s;
-      transition: color 0.3s ease 0s;
       background: transparent;
-      &:after {
-        -webkit-backface-visibility: hidden;
-        -moz-backface-visibility: hidden;
+      transition: color 0.3s ease 0s;
+
+      &::after {
+        position: absolute;
+        z-index: -1;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: calc(100% + 1px);
+        border: 1px solid #10cfbd;
         backface-visibility: hidden;
         background: none repeat scroll 0 0 #10cfbd;
-        border: 1px solid #10cfbd;
         content: "";
-        height: calc(100% + 1px);
-        left: 0;
-        position: absolute;
-        top: 0;
-        -webkit-transform: translate3d(0, 100%, 0px);
-        transform: translate3d(0, 100%, 0px);
-        -webkit-transition: -webkit-transform 0.3s ease 0s;
+        transform: translate3d(0, 100%, 0);
```

Comment:
Looks like another instance where `translate3d` could be swapped for just `translate`.